lectures/cagan_ree.md

```diff
@@ -236,6 +236,7 @@ As usual, we'll start by importing some Python modules.
 import numpy as np
 from collections import namedtuple
 import matplotlib.pyplot as plt
+plt.rcParams['figure.dpi'] = 200
```

Comment:
@HumphreyYang @mmcky Is there a reason that this particular lecture needs high dpi figures?  If not, let's get rid of this line.  If yes, would it benefit other lectures?  And if so, can we put this in a global config, rather than at the start of each lecture?

Reply:
Many thanks @jstac. 

I experimented with this lecture with the default dpi. The plots are blurry as they are long plots. I think not every lecture has this specific problem, so I used a global setting in this lecture.

Reply:
@jstac there are some rare moments when default dpi doesn't quite cut it from `matplotlib`. I have seen this in some other instances where we specify `dpi=200` in the figure creation. I think that is the best way to specify it (on a figure by figure basis) as not all figures will need it and it is clear what that code does. 

I agree with @jstac that we generally want to avoid the rc stuff as its scary :-)

Reply:
Many thanks @mmcky. I remember there was a discussion on this and we settled on using the global setting for simplicity if the setting is to be deployed on all graphs, but please correct me if I were wrong.

Reply:
@HumphreyYang are all the charts in this lecture large? In my view `plt.subplots(dpi=200)` is preferred but let leave this to @jstac to make the final call. That way it is consistent with limited use of figsize. 

Once we all agree I will update https://manual.quantecon.org/styleguide/figures.html

Reply:
> @HumphreyYang are all the charts in this lecture large? In my view plt.subplots(dpi=200) is preferred but let leave this to @jstac to make the final call. That way it is consistent with limited use of figsize.

That makes sense @mmcky .  Let's do as you suggest.

Reply:
Many thanks @jstac and @mmcky. I will revert my changes on `dpi` settings.